aep/general/example.oas.yaml

```diff
@@ -317,13 +251,72 @@
                             { '$ref': '#/components/schemas/publisher' },
                         },
                     },
+                  'required': true,
+                },
+            },
+          'delete':
+            {
+              'summary': '',
```

Comment:
totally, I think it might just be an issue with the linter... 

I'll take a look at whether we can add something there. the proto one has comments everywhere, so I don't see much of a reason to not do the same with OAS.

Reply:
removed! thanks for catching that.